react/features/base/sounds/actions.js

```diff
@@ -103,11 +86,14 @@ export function stopSound(soundId: string): Object {
  * @param {string} soundName - The name of bundled audio file that will be
  * associated with the given {@code soundId}.
  * @param {Object} options - Optional paramaters.
+ * @param {boolean} options.loop - True in order to loop the sound.
  * @returns {{
  *     type: REGISTER_SOUND,
  *     soundId: string,
  *     src: string,
- *     options: Object
+ *     options: {
+ *          loop: boolean
```

Comment:
hmm you could have defined a type and use it will flow like I did for AudioElement